sw/device/tests/sim_dv/alert_handler_escalation.c

```diff
@@ -0,0 +1,162 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
@jdonjdon 
@matutem 

you guys have worked a lot with alert based tests recently, does this look reasonable to you?

Reply:
> @jdonjdon @matutem
> 
> you guys have worked a lot with alert based tests recently, does this look reasonable to you?

LGTM with a two comments.